scripts/publish/aws-s3.sh

```diff
@@ -31,20 +31,23 @@ function usage() {
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
   echo "    -p <product name>"
+  echo "    -k <S3 key prefix>"
```

Comment:
> and fall beyond the scope of this PR

Yeah, after we'd agreed what syntax to use, I was planning to submit a separate PR updating all the other scripts to the same 'standard' :)